maas/plugins/hp_monitoring.py

```diff
@@ -59,7 +59,7 @@ def get_controller_status():
 
 def get_controller_cache_status():
     return check_command(('hpssacli', 'ctrl', 'all', 'show', 'status'),
-                         'Cache Status', 'OK')
+                         'Cache Status', ('OK', 'Not Configured'))
```

Comment:
I have not come across a unconfigured cache yet, we may remove that until we come across this